src/containers/forms/PlanForm/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,3225 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`containers/forms/PlanForm renders activity fields correctly: FI activities[0].actionCode field 1`] = `
```

Comment:
Wicked, and this snapshot syntax is generated like this?
```
...
expect(toJson(wrapper.find(`#activities-${i}-actionTitle input`))).toMatchSnapshot(
   `${name} activities[${i}].actionTitle field`
);
...
```

Reply:
Yes